### ISSUE TITLE
Accessibility: Inconsistent identification of controls

### ISSUE THREAD

**Issue by joedolson** [Automattic/tour#33]
The 'Close' button and the "Dismiss the tour" link perform very similar actions, but using different models. The use of a link for 'Dismiss the tour' implies that this control will send the user elsewhere to do this action.

"Dismiss the tour" should be a button.
